packages/amplify_datastore/example/lib/main.dart

```diff
@@ -27,7 +27,7 @@ import 'package:amplify_flutter/amplify.dart';
 import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/services.dart';
-import 'amplifyconfiguration.dart';
+//import 'amplifyconfiguration.dart';
```

Comment:
Why are we doing this?

Reply:
ah thanks - removed that for testing and forgot to change it back.